source/microbit/microbitconstimagetuples.c

```diff
@@ -0,0 +1,58 @@
+/*
+ * The MIT License (MIT)
```

Comment:
Someone needs to "own" it so they can decide on how the code is used.  I'm ok to give it to the PSF, but someone from there would need to approve it.

Otherwise you could make this file public domain, but it might be a bit strange to have some files MIT, some PD...